ui/src/pages/Club/styles.module.scss

```diff
@@ -0,0 +1,99 @@
+@import '../../index.scss';
+.container {
+  background-color: #fff;
+  margin: 1rem;
+  border-radius: 32px;
+  .h2 {
+    color: #889fb4;
+    font-size: 1.2rem;
+    margin-bottom: 1.5rem;
+  }
+  .p {
+    color: #304455;
+  }
+  .header {
+    height: 17.5rem;
+    background-color: red;
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    border-radius: 32px 32px 0 0;
+    position: relative;
+    .headerContent {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      border-radius: 32px 32px 0 0;
+      background-color: rgba(0, 0, 0, 0.65);
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      .logo {
+        width: 5rem;
+        height: 5rem;
+        border-radius: 50%;
+      }
+      .clubName {
+        font-size: clamp(1rem, 7vw, 2.25rem);
+        font-weight: 600;
+        color: #fff;
+      }
+      .registerBtn {
+        all: unset;
+        background-color: #d81f26;
+        color: #fff;
+        border-radius: 15px;
+        font-weight: 600;
+        width: 10rem;
+        height: 2.5rem;
+        text-align: center;
+        cursor: pointer;
+        transition: all 0.2s ease-in-out;
+        &:hover {
+          transform: scale(1.05);
+        }
+      }
+    }
+  }
+  .pageContent {
+    display: flex;
+    padding: 2rem;
+    margin: 0 1rem;
+
+    .sidebar {
+      width: 30%;
+      margin-right: 1rem;
+    }
+    .main {
+      width: 70%;
+      .about {
+        .description {
+          color: #304455;
```

Comment:
Try to use [css-variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) wherever possible, so we can have consistent styling. You can add new variables [here](https://github.com/BitByte-TPC/gymkhana/blob/a50f0acc6c750ac2d65a3b2ec2fa89a1044c52ac/ui/src/index.scss#L17).

Reply:
are you talking about color?

Reply:
yes